packages/pds/src/account-manager/index.ts

```diff
@@ -327,7 +327,7 @@ export class AccountManager {
       await this.db.transaction((dbTxn) =>
         Promise.all([
           account.updateEmail(dbTxn, did, email),
-          emailToken.deleteEmailToken(dbTxn, did, 'update_email'),
+          emailToken.deleteAllEmailTokens(dbTxn, did),
```

Comment:
Do we want to run this even when there is no `token`?

Reply:
Yup I believe so

becuase we want to clear _all_ tokens regardless since they were associated with the old email

Reply:
Ah, I actually meant that it wasn't being run in that case and I figured we might want to!